config.go

```diff
@@ -9,8 +9,9 @@ const (
 )
 
 type config struct {
-	StorageType string `envconfig:"STORAGE_ADAPTER" default:"memory"`
-	NumLines    int    `envconfig:"NUMBER_OF_LINES" default:"1000"`
+	StorageType    string `envconfig:"STORAGE_ADAPTER" default:"memory"`
+	NumLines       int    `envconfig:"NUMBER_OF_LINES" default:"1000"`
+	AggregatorType string `envconfig:"AGGREGATOR_TYPE" default:"nsq"`
```

Comment:
As we discussed out of band, those are in `log/config.go`-- just trying to keep those settings closer to where they are used.